docs/_data/components/schemas/array-max-monitored-elements.yaml

```diff
@@ -122,20 +118,56 @@ forecast-proposal-status:
             # the client submitted.
             $ref: ./resource-id.yaml
 
+  required:
+    - incomplete-obligation-count
+    - incomplete-obligations
+    - invalid-proposal-count
+    - proposal-validation-errors
+
+
+forecast-proposal-status:
+  type: object
+  description: |
+    Status of the rating provider's forecast proposal relative to the current 
+    forecast window.  Includes any validation errors encountered while processing proposals.  
+  properties: 
+    <<: *common-proposal-status-props
+    forecast-provider:
```

Comment:
Is it worth making a breaking change to `ratings-provider` to harmonize with the RT proposal?

Reply:
We decided no.  Just marking it down as I clean this up.